src/AM.Condo.Abstractions/AM.Condo.Abstractions.csproj

```diff
@@ -5,9 +5,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Runtime.Extensions" Version="[4.3.0, 5.0.0)" />
-    <PackageReference Include="System.Collections" Version="[4.3.0, 5.0.0)" />
-    <PackageReference Include="NuGet.Versioning" Version="[4.4.0, 5.0.0)" />
+    <PackageReference Include="System.Runtime.Extensions" Version="4.3.0" />
```

Comment:
@dmccaffery any reason why we are version locking now?

Reply:
@sjk07 it doesn't actually version lock anymore; binding redirects were reintroduced and are autogenerated. scoping ranges doesn't have much effect anymore.

Reply:
The other reason is: we control the environment in which condo runs, now -- within docker; no reason not to pin the versions if we can -- avoids nuance bugs (although binding redirects will happen anyway)

Reply:
sounds magical.